src/components/activities/adminActivity/AdminActivity.module.scss

```diff
@@ -14,7 +14,7 @@
   border-radius: 8px;
   flex-grow: 1;
   margin: 30px 160px;
-  align-self: center;
+  // align-self: center;

```

Comment:
![master-yoda-star-wars](https://github.com/informatici/openhospital-ui/assets/597067/0f612ec7-6a9e-4825-930f-fdbc65fe70c9)

Reply:
I commented out this rule because it's causing the layout to shrink or grow depending on the content. I didn't remove it completely to keep a record, since I wasn't the one who initially defined this rule.

Reply:
Please don't worry about keeping a record, `git blame` can be used if necessary.
Please don't hesitate, [be bold](https://en.wikipedia.org/wiki/Wikipedia:Be_bold). It's open source software, if you see something wrong, please fix it!

If you think that this rule is incorrect, please remove it.
If you think that it's correct, please leave it.

If you make a change, test.
If you doubt, and don't understand why it was there before, please test even more :-)